commcare_connect/templates/opportunity/opportunity_detail.html

```diff
@@ -45,13 +45,6 @@ <h1 class="mb-0">{{ object.name }}</h1>
             </a>
           </li>
           {% if object.managed and request.org_membership.is_program_manager or request.user.is_superuser %}
-          <li>
-            <a class="dropdown-item"
-               href="{% url "program:opportunity_application_list" org_slug=request.org.slug pk=opportunity.managedopportunity.program.id opp_id=opportunity.pk %}">
```

Comment:
Was it just a miss that this was not removed in the PR that removed this view?

Reply:
This was removed in the PR that removed the view, but we had a lot of PR's that were based on each other and that led to me adding this back into one of the PRs by mistake.